cdc_replication_test.py

```diff
@@ -41,7 +39,7 @@ def mode_str(mode: Mode) -> str:
     return {
         Mode.DELTA: 'delta',
         Mode.PREIMAGE: 'preimage',
-        Mode.POSTIMAGE: 'postimage'
+        Mode.POSTIMAGE: 'postimage',
```

Comment:
can we make ruff behave like pylint and ignore these?

Reply:
We could, but those are auto fixable, and something @vponomaryov for example was asking in lots of PRs